### PR TITLE
feat: support mse gateway policy

### PR DESCRIPTION
**Pull request by wangzhuzhen**
Add document about how to use and set gateway policy when use MSE Gateway in Erda。

### CODE REVIEW

**Review by sfwn — APPROVED** [review]

File: docs/next/manual/msp/guides/apigw/mse_policy.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+##### 正常返回:
+
+```bash
+curl -v 'http://test-gateway.mse-daily.terminus.io/testmse/test/policy' \
+    -H 'Cookie: uc-token=1ba855fcb59caa9d82370495d; OPENAPISESSION=937ec28c-b3a7-4f64-95ad-4fa0e4e19ad3' \
+    -H 'Csrf-Token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJ1cHN0cmVhbSIsImV4cCI6IjE2ODQ4MzU1ODgiLCJpYXQiOiIxNjg0ODMzNzg4IiwiaXNzIjoiRXJkYS1BUEktR2F0ZXdheSIsImp0aSI6IjFiYTg1NWZjYjU5Y2FhOWQ4MjM3MDQ5NWQiLCJuYmYiOiIxNjg0ODMzNzg4Iiwic3ViIjoiZnJvbnRlbmQifQ.afZWTG5s0b7l_-9e1-iYEe8W5xMEIC1kRgCSe607DZs' \
+    --compressed \
+    --insecure
+*   Trying 116.62.243.103:80...
```
Comment (sfwn) [inline docs/next/manual/msp/guides/apigw/mse_policy.md:172]: Sensitive information needs to be desensitized.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: docs/next/manual/msp/guides/apigw/mse_policy.md (hunk 1 above)
Comment (wangzhuzhen) [reply docs/next/manual/msp/guides/apigw/mse_policy.md:172]: Masked.

**Review by sfwn — APPROVED** [review]

**Pull request merged**